frontend/shared/components/src/filters/GroupUIFilter.ts

```diff
@@ -78,7 +78,7 @@ export class GroupUIFilter extends UIFilter {
         })
     }
 
-    get styledDescription() {
+    override get styledDescription(): StyledDescription {
```

Comment:
Probeer wijzigingen wel wat op te splitsen in de toekomst, de PR hier had beter opgesplitst geweest in één voor de filter ondersteuning op vlagmomenten, één voor de wijzigingen van inverted filters + dropdowns, en één voor de visuele wijziging van de setups steps in het administratieportaal — dat maakt het makkelijker om te zien wat er veranderd is en om in de toekomst fouten te debuggen